### ISSUE TITLE
 Invalid arguments supplied to oneOf, expected an array, got 10 arguments

### ISSUE THREAD

**Issue by JonThom** [thedirtyfew/dash-extensions#39]
This occurs twice in Burger.react.js. 
Solution: wrap the arguments to oneOf in an array.

**Comment by emilhe** [comment]
I have noticed it myself, but I couldn't find the property causing it. Can you see which one it is?

**Comment by JonThom** [comment]
I think wrapping the types in an array like this would fix it:

```
position: PropTypes.oneOf(["left", "right"]),

effect: PropTypes.oneOf(["slide", "stack", "elastic", "bubble", "push", "pushRotate", "scaleDown",
        "scaleRotate", "fallDown", "reveal"]),
```

**Comment by emilhe** [comment]
Ah, yes of course, thanks! I have pushed the fix in version 0.0.42,

https://pypi.org/project/dash-extensions/0.0.42/
